File: Kattis/Python/Egypt.py
done = False

while not done:
    abc = input().split()
    sides = []

    for x in abc:
        sides.append(int(x))

    sides.sort()

    a = sides[0]
    b = sides[1]
    c = sides[2]

    if a != 0 and b != 0 and c != 0:
        if a ** 2 + b ** 2 == c ** 2:
            print("right")
        else:
            print("wrong")
    else:
        done = True

File: Kattis/Java/Egypt.java
import java.util.Scanner;
import java.util.Arrays;

public class Egypt{

	public static void main(String[] args){
		Scanner sc = new Scanner(System.in);

		String[] input;
		int a, b, c;

		do{
			 input = sc.nextLine().split(" ");

			 a = Integer.parseInt(input[0]);
			 b = Integer.parseInt(input[1]);
			 c = Integer.parseInt(input[2]);

			 if(a != 0 && b != 0 && c != 0){
			 	if(a * a + b * b == c * c)
			 		System.out.println("right");
			 	else System.out.println("wrong");
			 }

		} while( a != 0 && b != 0 && c != 0);

		sc.close();
	}
} 